stack.yaml

```diff
@@ -7,6 +7,7 @@ packages:
 extra-deps:
   - nonempty-containers-0.3.4.0
   - nonempty-vector-0.2.0.2
+  - nothunks-0.1.1.0
```

Comment:
Are we going to remove the `stack.yaml` from this repo too? What about `snapshot.yaml`?

Reply:
I think we will, but let's do that in another PR.